### PR TITLE
Create a general page guide 

### PR DESCRIPTION
**Pull request by bedroesb**
How do we want to link towards this page in the contribute?

### CODE REVIEW

File: page_guide.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  pathogen_characterisation: [<!---REPLACE THIS with the page ID of the pathogen_characterisation pages that you want to list here as related pages--->]
+training:
+  - name:
+    registry:
+    url:
+---
+
+## Concrete topic 1 <!-- Example: Metadata harmonisation-->
+
+Short explanation of what this topic is about with an emphasis on infectious diseases.
```
Comment (rabuono) [inline page_guide.md:20]:
```suggestion
Short explanation of what this topic is about and why it is important, with an emphasis on infectious diseases.
```

File: page_guide.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+### Solutions <!-- do not delete this heading and write your text below it -->
+
+By using bullet point style as much as possible, try to describe when, why and for what is best to use a specific tool or resource. 
+Avoid making long list of links to tools and resources.
```
Comment (rabuono) [inline page_guide.md:26]:
```suggestion
Avoid making long lists of links to tools and resources.
```

**Review by rabuono — COMMENTED** [review]
Added two minor edits.
Looks like a good basic start to me.

**Comment by bedroesb** [comment]
@rabuono do we wait for at least one extra approval?

File: page_guide.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: <!---Page title--->
+description: <!---Short description of the page--->
+contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
+page_id: <!---REPLACE THIS with a shortened page name, with small letters and spaces, or an acronym in capital and small letters--->
```
Comment (LianeHughes) [inline page_guide.md:5]:
```suggestion
page_id: <!---REPLACE THIS with a shortened page name, with lower case letters and spaces, or an acronym in upper and lower case letters--->
```

File: page_guide.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+related_pages: 
+  showcase: [<!---REPLACE THIS with the page ID of the showcase pages that you want to list here as related pages--->]
```
Comment (LianeHughes) [inline page_guide.md:7]:
```suggestion
  showcase: [<!---REPLACE THIS with the page IDs of the showcase pages that you want to list here as related pages--->]
```

File: page_guide.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  human_biomolecular_data: [<!---REPLACE THIS with the page ID of the human_biomolecular_data pages that you want to list here as related pages--->]
```
Comment (LianeHughes) [inline page_guide.md:8]:
```suggestion
  human_biomolecular_data: [<!---REPLACE THIS with the page IDs of the human_biomolecular_data pages that you want to list here as related pages--->]
```

File: page_guide.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  human_clinical_and_health_data: [<!---REPLACE THIS with the page ID of the human_clinical_and_health_data pages that you want to list here as related pages--->]
```
Comment (LianeHughes) [inline page_guide.md:9]:
```suggestion
  human_clinical_and_health_data: [<!---REPLACE THIS with the page IDs of the human_clinical_and_health_data pages that you want to list here as related pages--->]
```

File: page_guide.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  social_and_economic_impact: [<!---REPLACE THIS with the page ID of the social_and_economic_impact pages that you want to list here as related pages--->]
```
Comment (LianeHughes) [inline page_guide.md:10]:
```suggestion
  social_and_economic_impact: [<!---REPLACE THIS with the page IDs of the social_and_economic_impact pages that you want to list here as related pages--->]
```

File: page_guide.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  pathogen_characterisation: [<!---REPLACE THIS with the page ID of the pathogen_characterisation pages that you want to list here as related pages--->]
```
Comment (LianeHughes) [inline page_guide.md:11]:
```suggestion
  pathogen_characterisation: [<!---REPLACE THIS with the page IDs of the pathogen_characterisation pages that you want to list here as related pages--->]
```

File: page_guide.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Concrete topic 1 <!-- Example: Metadata harmonisation-->
+
+Short explanation of what this topic is about and why it is important, with an emphasis on infectious diseases.
+
+
+### Solutions <!-- do not delete this heading and write your text below it -->
+
+By using bullet point style as much as possible, try to describe when, why and for what is best to use a specific tool or resource.
```
Comment (LianeHughes) [inline page_guide.md:25]:
```suggestion
Using bullet point list style as much as possible, describe when, why and for what purpose a specific tool or resource should be used. 
```

**Review by LianeHughes — COMMENTED** [review]
Suggested a few very little changes. I wonder if it is worth mentioning that the page IDs listed should also be in comma separated lists? (I realise you might not always have multiple pages, I was just thinking that we mentioned the comma separation elsewhere.

I hope I have done this review as we're supposed to. Apologies if not. Please tell me if I should have done something differently here.

**Comment by bedroesb** [comment]
> Suggested a few very little changes. I wonder if it is worth mentioning that the page IDs listed should also be in comma separated lists? (I realise you might not always have multiple pages, I was just thinking that we mentioned the comma separation elsewhere.
> 
> I hope I have done this review as we're supposed to. Apologies if not. Please tell me if I should have done something differently here.

You did great @LianeHughes !

**Review by rabuono — APPROVED** [review]
To me, this looks good for merging and trying it out.

**Pull request merged**